content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md

```diff
@@ -98,39 +106,56 @@ Commits can be hidden from the project table in the following ways:
   commits that do not add much value in your project. To hide a commit or
   branch, click on the 3-dot menu next to the commit or branch name and click on
   `Hide commit` or `Hide branch`.
+
+  ![Hide commit](https://static.iterative.ai/img/studio/hide_commit.png)
+
 - **Unhide commits:** You can unhide commits as needed, so that you don't lose
   any experimentation history. To display all hidden commits, click on the
-  `Show hidden commits` toggle (refer [the above gif](#display-preferences)).
-  This will display all hidden commits, with a `hidden` (closed eye) indicator.
+  `Show hidden commits` toggle (refer [filters](#filters)). This will display
+  all hidden commits, with a `hidden` (closed eye) indicator.
+
+  ![Hidden commit indicator](https://static.iterative.ai/img/studio/hidden_commit_indicator.png)
+
   To unhide any commit, click on the 3-dot menu for that commit and click on
   `Show commit`.
 
-### Selected only:
+  ![Show hidden commit](https://static.iterative.ai/img/studio/show_hidden_commit.png)
 
-Toggle between showing and hiding experiments that you have not selected.
+### Delta mode
 
-### Delta mode:
+You can display either the absolute values of the columns (metrics,
+hyperparameters, etc) or their delta (difference) from the baseline row. To
+toggle between these two options, use the `Delta mode` button.
```

Comment:
I would mention that this is the behavior for numerical values (how would other values like text be treated?)

Reply:
> Can you pls confirm this?

It works for metrics, models and files, e.g.,

<img width="681" alt="Screenshot 2023-08-14 at 21 40 49" src="https://github.com/iterative/dvc.org/assets/28868855/4439f397-4c02-4d5c-ade5-ecbbc573c7b3">

> Also, why doesn't Delta mode work on hyper-parameters - is it intended behavior or a bug?

Neither, I believe. Historically, the params were just strings, and at some point we started to distinguish between different types. We've supported all types, but just forgot to implement delta mode for them. Feel free to file an issue, no tech blockers exist behind the numerical params delta mode support.

Reply:
I've updated the text [here](https://github.com/iterative/dvc.org/pull/4758/commits/11b0bd52b4edfe3e5a74f40be650e5307807db8a).